ibu-imager/ostreeclient/rpmostreeclient.go

```diff
@@ -112,3 +115,76 @@ func (c *Client) QueryStatus() (*Status, error) {
 
 	return &q, nil
 }
+
+// TODO: replace with https://github.com/coreos/rpmostree-client-go
+//       when https://github.com/coreos/rpmostree-client-go/issues/23 resolves
+
+// QueryStatusChroot uses chroot to loads the current system state
+func QueryStatusChroot(rootPath string) (*Status, error) {
+	cmd := exec.Command("/usr/bin/env", "--", "bash", "-c", "rpm-ostree status --json")
```

Comment:
This logic should go to https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/ops/execute.go 
In this case we can provide interface and have this code testable